annotation/tests/test_jobs.py

```diff
@@ -6,35 +6,28 @@
 
 import pytest
 
+import annotation.jobs.services
```

Comment:
better import it like that - 
```suggestion
from annotation.jobs import services
```
if you use other services too and name collision exists, import jobs